es6/tests/utils.js

```diff
@@ -543,6 +543,10 @@ function createDoc(name) {
 	return loadDocument(name, documentCache[name].loadedContent);
 }
 
+function createDocV4(zip, { modules }) {
```

Comment:
Can you reuse the same level of abstraction as in createDoc ? 

eg instead of `createDocV4(zip, { modules })`, have : `createDocV4(name, { modules})`

Reply:
Changing the test to validate for the calling of module will do the trick for this one also. 👍

Reply:
Have changed the method signature to the suggested.